### ISSUE TITLE
Devstack Install

### ISSUE THREAD

**Issue by alta3** [alta3/openstack-labs#1]
Students should not have to sit at a screen and watch stuff scroll unless there is some powerful learning reason to do so, and I don't think there is, so it may be more interesting to teach how to use an ansible script to install the software.  Just an idea.....  I have no solid convictions at this point, but I would love a valid reason to teach ansible as soon as possible.  Maybe Labs 1 + should be github and ansible

**Comment by bryfry** [comment]
Checkout [Installing Ansible](http://docs.ansible.com/ansible/intro_installation.html).  Not necessarily  simple if you are assuming students have Win,OSX,Linux laptops that they are trying to get this installed on.

One solution could be a t2.micro server that all labs are done through.  So student laptop -> lab-host -> openstack

**Comment by alta3** [comment]
Yes, I was thinking all I can count on the student providing is a browser and SSH. So we have to provide the landing spot for running ansible, so the t2.micro was exactly what I was thinking about.  They SSH into their micro which has ansible already installed so that we can demo "see it works". So first the students learn how to be operators.  This machine would already be provided for them, sort of their home base. From this machine, they will run ansible playbooks, then learn how to modify and create their own playbooks etc.

**Comment by alta3** [comment]
Also, a diagram of their environment is something Zach and I can do, ASCII art or a scribbled diagram is all we need to document the system for "operations training" for lack of a better word.

**Comment by bryfry** [comment]
```
                                           ┌──────────────┐
                                           │  OpenStack1  │
                                       ┌──▶│  t2.medium   │
┌──────────────┐     ┌──────────────┐  │   │              │
│              │     │    Jumper    │  │   └──────────────┘
│Student Laptop│────▶│  (t2.micro)  │──┤                   
│              │     │              │  │   ┌──────────────┐
└──────────────┘     └──────────────┘  │   │  OpenStack2  │
                                       └──▶│  t2.medium   │
                                           │              │
                                           └──────────────┘
```

**Comment by alta3** [comment]
Looks exactly like I was thinking.  This is a two node system.  We may need
a three node system if that's what it takes to demo a network node.  So
that would be controller...network...compute nodes.  Is this necessary?  No
idea just reading about it in examples.

**Comment by bryfry** [comment]
Updated OpenStack to DevStack.  Still haven't gotten to the RDO Openstack portion, those may be additional nodes.

**Comment by bryfry** [comment]
```
                                           ┌──────────────┐
                                           │  DevStack1   │
                                       ┌──▶│  m4.2xlarge  │
                                       │   │    Ubuntu    │
                                       │   └──────────────┘
                                       │                   
┌──────────────┐     ┌──────────────┐  │   ┌──────────────┐
│              │     │    Jumper    │  │   │  DevStack2   │
│Student Laptop│────▶│  (t2.micro)  │──┼──▶│  m4.2xlarge  │
│              │     │              │  │   │    Ubuntu    │
└──────────────┘     └──────────────┘  │   └──────────────┘
                                       │                   
                                       │   ┌──────────────┐
                                       │   │RDO OpenStack │
                                       └──▶│  m4.2xlarge  │
                                           │    CentOS    │
                                           └──────────────┘
```

Updated graph to show RDO.  Also updated the instance size, we can get an m4.2xlarge spot instance for the same price (~0.05 per hour) as a t2.medium and get 8 cores vs 2.  I'm testing on both sizes though.
